### ISSUE TITLE
Implement MacType font-rendering within Chromium

### ISSUE THREAD

**Issue by GTANAdam** [GTANAdam/GDIChromium#6]
https://github.com/snowie2000/mactype

**Comment by ufuksarp** [comment]
What does it mean to implement MacType within Chromium? Doesn't implementing GDI back into Chromium do that automatically?

**Comment by GTANAdam** [comment]
> What does it mean to implement MacType within Chromium? Doesn't implementing GDI back into Chromium do that automatically?

That doesn't automatically include it. GDI only allows MacType (externally) to function properly.
MacType utilizes other sampling techniques and mainly the [FreeType font rasterization engine](https://www.freetype.org/).
I will first be including MacType (internally) within the binary and afterwards, replacing the default font rendering engine with the mentioned above, FreeType.

**Comment by ufuksarp** [comment]
> That doesn't automatically include it. GDI only allows MacType (externally) to function properly. MacType utilizes other sampling techniques and mainly the [FreeType font rasterization engine](https://www.freetype.org/). I will first be including MacType (internally) within the binary and afterwards, replacing the default font rendering engine with the mentioned above, FreeType.

Yeah, I should have worded it better. I guess my question really was "why implement MacType into the browser". Why not just let MacType do the work as it does now with the likes of Cent Browser?

Furthermore, if you think this is the way to go, will it be possible to customize everything the way you can with the current MacType **ini** files?

**Comment by GTANAdam** [comment]
> Yeah, I should have worded it better. I guess my question really was "why implement MacType into the browser". Why not just let MacType do the work as it does now with the likes of Cent Browser?
> 
> Furthermore, if you think this is the way to go, will it be possible to customize everything the way you can with the current MacType **ini** files?

It's much easier to install a working browser with better font rendering than installing separate applications in order to achieve that, much better user experience as a result. additionally, it should perform much better in terms of performance.

Sure, that's feasible of course.
